### DISCUSSION TITLE
How define Blackhole as Input Device Only ?

### DISCUSSION THREAD

**Discussion by mohshaat1990** [ExistentialAudio/BlackHole#415] [Q&A]
thank you for this great project. I have a question about  audio server Plugin How to define it as Input only because Blackhole is appear in both sound Output & input list . I want to make it appear it in Input only How to do that ? if you know a document could you direct me to it? because I tried to even read apple Null Driver and didn't reach to any solution

**Comment by phpnukes** [comment]
I get into a right pickle with my Audio midi setup using the latest Mac.
From memory ignore the output side when you create a device option select aggregate select blackhole ch2 or c16 or soundflower what ever you using. I used audacity so reboot what software you are using. Then select in preferences in this software audio devices.
